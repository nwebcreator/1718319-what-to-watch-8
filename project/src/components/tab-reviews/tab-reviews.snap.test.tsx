import { render } from '@testing-library/react';
import TabReviews from './tab-reviews';

describe('Component: TabReviews', () => {
  it('should render correctly', () => {
    const { container } = render(<TabReviews reviews={[]} />);
    expect(container).toMatchSnapshot();
  });
});
