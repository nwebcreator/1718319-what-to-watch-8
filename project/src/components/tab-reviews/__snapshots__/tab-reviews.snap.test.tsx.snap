// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: TabReviews should render correctly 1`] = `
<div>
  <div
    class="film-card__reviews film-card__row"
  >
    <div
      class="film-card__reviews-col"
    />
    <div
      class="film-card__reviews-col"
    />
  </div>
</div>
`;
